fake_grasp_pose_publisher:
  ros__parameters:
    frame_id: camera_frame
    grasp_pose: [0.0126, 0.2322, 0.442,
                 0.0, 0.0, 0.9997, 0.0250]
    
    object_pose: [0.0128, 0.2330, 0.543,
                  0.0, 0.0, 0.9997, 0.0250]

    object_dimensions: [0.087, 0.167, 0.023]

    delay: 1.5
